openy_activity_finder.install

```diff
@@ -223,7 +223,7 @@ function openy_activity_finder_update_8402() {
 /**
  * Update configs for search index.
  */
-function openy_activity_finder_update_8403() {
+function openy_activity_finder_update_8404() {
```

Comment:
instead of renaming old function, let's introduce a new one and call old from it @froboy

Reply:
Great suggestion @podarok. I elaborated on the existing function to make it a little more flexible.